src/components/add-tag-dialog/add-tag-dialog.js

```diff
@@ -19,6 +20,9 @@ export class AddTagDialogComponent {
     );
     this.tagTextField = this.mountPoint.querySelector(".add-tag-dialog__tag");
     this.tagNameInput = this.mountPoint.querySelector(".mdc-text-field__input");
```

Comment:
It is not yours, but here it us not good idea to match element with mdc class. For our code we use own classes